frontends/ol-widgets/src/factories.ts

```diff
@@ -1,5 +1,6 @@
 import { faker } from "@faker-js/faker"
```

Comment:
```suggestion
import { faker } from "@faker-js/faker/locale/en"
```

As you are not changing the locale **at runtime**, just use the `en` locale